pages/[resourceType]/index.tsx

```diff
@@ -46,7 +66,28 @@ function ResourceTypeIDs() {
       <Loader color="cyan"></Loader>
     </Center>
   ) : !fetchingError && pageBody ? ( //if http request was successful, ResourceID component is returned
-    <ResourceIDs jsonBody={pageBody}></ResourceIDs>
+    <div>
+      <div
+        style={{
+          paddingRight: "20px",
+        }}
+      >
+        <Link href={`${resourceType}/create`} key={"create" + resourceType} passHref>
+          <Button
+            color="cyan"
```

Comment:
if we're using `passHref` above, should we also add the following prop as per the Mantine docs for convention of usage with nextjs?

```suggestion
            component="a"
            color="cyan"
```